packages/venia-concept/src/components/MiniCart/miniCart.css

```diff
@@ -1,11 +1,13 @@
 .root {
+    --minicart-height: 100vh;
+    --minicart-header-height: 3.5rem;
```

Comment:
[paymentsForm.css](https://github.com/magento-research/pwa-studio/blob/supernova/payments/packages/venia-concept/src/components/Checkout/paymentsForm.css#L7-L11) needed to know them in order to set a `max-height` so that it could scroll.

If there's a better way to do this definitely let me know.